spec/features/budgets/votes_spec.rb

```diff
@@ -173,4 +173,29 @@
 
     end
   end
+
+
+  feature 'Approval Voting' do
```

Comment:
It seems there is a related failing spec  https://travis-ci.org/consul/consul/jobs/482549751#L1887

Reply:
I think there were tests to be written. I only finished some of the comments of the previous PR.

Reply:
I will update the PR with the tests taken care of.

Reply:
Thanks for the increased test coverage 👏